crates/offchain-db/src/tables.rs

```diff
@@ -0,0 +1,54 @@
+use std::fmt;
+
+use crate::utils::get_db_extension;
+
+pub enum Tables {
+    /// string version is sequencer_commitment
+    #[allow(dead_code)]
+    SequencerCommitment,
+}
+
+// impl to_string for tables
+impl fmt::Display for Tables {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            Tables::SequencerCommitment => write!(f, "sequencer_commitment"),
+        }
+    }
+}
+
+#[derive(Debug, Clone)]
+pub struct DbSequencerCommitment {
+    /// Hex encoded L1 transaction ID
+    pub l1_tx_id: String,
+    pub l1_start_height: u32,
+    pub l1_end_height: u32,
+    /// Hex encoded L1 start hash
+    pub l1_start_hash: String,
+    /// Hex encoded L1 end hash
+    pub l1_end_hash: String,
+    pub l2_start_height: u64,
+    pub l2_end_height: u64,
+    /// Hex encoded merkle root of soft confirmation hashes
+    pub merkle_root: String,
```

Comment:
lets handle hashes as byte arrays

Reply:
we can fetch as byte array on nodes or sequencer and fetch with `select encode(..., 'hex') from ..` in api